include/eve/concept/invocable.hpp

```diff
@@ -82,4 +83,15 @@ concept invocable = requires(F&& f, Args&&...args)
 {
   typename detail::get_invoke_result<F&&, Args&&...>::type;
 };
+
+  //================================================================================================
+  //! @ingroup simd_concepts
+  //! @brief std::predicate but doesn't require regularity
+  //!
+  //! @tparam T
+  //================================================================================================
+  template <typename T, typename ... Args>
+  concept irregular_predicate = requires (T&& f, Args&&... args) {
```

Comment:
why not relaxed_predicate ?

Reply:
Well, std::predicate is not working for me, because it requires `regular`.
so this is `irregular` :) I think it makes sense

Reply:
We have relaxed_range hence the idea. But I guess this is ok too.

Reply:
I understand yes. But I don't like it.

Tbh - I think - this should be predicate and the standard one should be regular_predicate. But we are here

Reply:
OK